HW2/P5/driver.py

```diff
@@ -16,6 +16,18 @@
 def randcolor():
     return np.random.uniform(0.0, 0.89, (3,)) + 0.1
 
+# morton ordering taken from wikipedia
+def cmp_zorder(a, b):
```

Comment:
Did you check that this worked correctly? I found that it failed for large matrices, i.e. matrices with a size greater than 2^16 or something. You can see this by visualizing a sorted matrix. I checked it out for our grid and it failed, I thought. Maybe you fixed it though!